themes/docsy/assets/scss/_custom.scss

```diff
@@ -903,6 +903,69 @@ body.tv-episode {
   margin-right: 10px;
 }
 
+.checklist-item {
+  padding: 2rem 2rem 1rem;
+  margin: 1.5rem 0;
+  display: flex;
+  border-left: 3px solid var(--MAIN-ANCHOR-color);
+}
+
+.checklist-item > input[type="checkbox"] {
+  margin-right: 1em;
+  margin-top: 5px;
+}
+
+.checklist-item h4 {
+  margin: 0;
+}
+
+table {
+  border: 1px solid #eaeaea;
+  table-layout: auto;
+  margin: 20px 0px;
+}
+th {
+  background: #ffffffda;
+  padding: 0.5rem;
+  color: black;
+  border: 1px solid black;
+}
+td {
+  padding: 0.5rem;
+  border: 1px solid #eaeaea;
+}
+
+div.aside-info > div.aside-title {
+  background: #1D428A;
+}
+
+div.aside-warning > div.aside-title {
+  background: #c53030;
+}
+
+div.aside-tip > div.aside-title {
+  background: #78BE20;
+}
+
+div.aside div.aside-title {
+    height: 30px;
+    /* To center the line vertically */
+    line-height: 30px;
+    color: #fff;
+    padding-left: 1em;
+    font-size: 1em;
+}
+
+div.aside div.aside-title i {
+    margin-right: 5px;
+}
+
+div.aside div.aside-content {
+    padding: 5px 20px;
+    background: #0f171c;
+    /* background-color: black; */
+}
```

Comment:
We will need to make sure to add light mode versions of these styles to the `light-theme.css` file.